extensions/wikia/WikiaInYourLang/WikiaInYourLangController.class.php

```diff
@@ -65,8 +69,17 @@ public function getNativeWikiaInfo() {
 						$oNativeWiki->city_url,
 						$oNativeWiki->city_title,
 					];
-
-					$sMessagesAry = $this->prepareMessage( $sTargetLanguage, $aMessageParams );
+					$isMainPageLink = true;
+					if ( $wgWikiaEnvironment !== WIKIA_ENV_DEV ) {
```

Comment:
@nandy-andy helped me around here. I vaguely remember that we did this because in dev box, there is not much article there so it is almost always does nothing. That is why we skip this for dev box. Correct me if I am wrong, @nandy-andy.